test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -416,7 +422,7 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, NoFinancialModelShading)
 		EXPECT_NEAR(calculated_value, annual_energy_expected[0], m_error_tolerance_hi);
```

Comment:
If this test is passing with the annual_energy_expected as on line 396,402 above, then the size_t get_row_index_for_input in common.cpp is probably okay! Great job!

Reply:
Haha thanks!